infra/gcp/clusters/.gitignore

```
@@ -1,4 +1,16 @@
 .terraform/
-*.tfstate*
+.terraform.lock.hcl
+
+# .tfstate files
+*.tfstate
+*.tfstate.*
+
+# Ignore override files as they are usually used to override resources locally and so
+# are not checked in
+override.tf
+override.tf.json
+*_override.tf
+*_override.tf.json
```

Unless there's a need to allow `foooverride.tfbar`?
```suggestion
*override.tf*
```

---

Fixed